src/components/Header/headerStyles.ts

```diff
@@ -59,6 +102,20 @@ export const Li = styled.li`
   }
 `;
 
+export const Burger = styled.div`
+  margin-right: 50px;
+  display: none;
+  background-image: url(${burger});
+  width: 40px;
+  height: 40px;
+  background-position: center;
+  background-size: 50px;
```

Comment:
add here cursor: pointer please

Reply:
I think it makes no sense to change the cursor for the menu of the mobile version